README.md

```diff
@@ -288,6 +288,78 @@ const req = {
 const sig = await client.sign(req)
 ```
 
+### 💾 Calldata Decoding
+
+> NOTE: all available calldata decoding options will be documented in this section. More may be added as time goes on.
+
+Certain transaction decoder types may support calldata decoding for request data. You can use this feature by including "calldata decoder data" (explained shortly) in a general signing request using the `decoder` request param:
+
+```
+req.decoder = <calldata decoder data>
+await client.sign(req);
+```
+
+If you include a valid calldata decoder, the appearance of the transaction's data on the user's Lattice should transform from a raw hex string to a markdown-style version which displays the function name, parameter names, and values.
+
+**Storing Calldata Decoders**
+
+Although not necessary, in certain situations it may be advantageous to pre-save decoders to the Lattice. One advantage is that if the decoder is saved, you do not need to include it in the transaction request, which frees up some space. Additionally, pre-saving data may unlock certain security features depending on the decoder type.
+
+You can use the following API:
+
+> Please see API docs for all options. Also see tests in `test/signing/evm.ts` for examples on usage.
+
+* `addDecoders`: Allows the user to add a series of calldata decoders for a specific decoder type (e.g. EVM). This will draw a screen on the user's Lattice which will require approval before returning success.
```

Comment:
I think the phrasing, "draw a screen", might be confusing for people who don't know how the hardware works. Maybe "this will prompt the user to approve on their Lattice" or something?

Reply:
[756bc00](https://github.com/GridPlus/gridplus-sdk/pull/353/commits/756bc001e62e4ba9fce75734645727253137ddc2)